suave/backends/redis_store_backend.go

```diff
@@ -0,0 +1,116 @@
+package backends
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	suave "github.com/ethereum/go-ethereum/suave/core"
+	"github.com/go-redis/redis/v8"
+)
+
+var ffStoreTTL = 24 * time.Hour
```

Comment:
`24` could be an env var 🤔

Reply:
I'd avoid giving a false sense of stability by making this an env var - this is subject to change soon